amplify/backend/api/rms/schema.graphql

```diff
@@ -1,5 +1,76 @@
-type Todo @model {
-  id: ID!
-  name: String!
-  description: String
+type Query {
+    allMain(count: Int, nextToken: String): PaginatedMains!
+    allItems(count: Int, nextToken: String): PaginatedItems!
+    allBatch(count: Int, nextToken: String): PaginatedBatches!
+    allTags(count: Int, nextToken: String): PaginatedTags!
+    allHistory(count: Int, nextToken: String): PaginatedHistories!
 }
+
+type Main @model @auth(rules: [{allow: public, operations: [read]}]) {
+    name: ID!
+    displayName: String!
+    description: String!
```

Comment:
This is optional

Reply:
Optional as in don't use "!" (nullable) or as in remove these fields?